ldap.go

```diff
@@ -166,14 +58,19 @@ func (x *ldapUserStore) GetLdapUsers() ([]AuthUser, error) {
 		"mail",
 		"mobile"}
 
-	search_request := ldap.NewSearchRequest(
+	searchRequest := ldap.NewSearchRequest(
 		"dc=imqs,dc=local",
 		ldap.ScopeWholeSubtree, ldap.DerefAlways, 0, 0, false,
 		"(objectclass=user)",
 		attributes,
 		nil)
 
-	sr, err := x.con.Search(search_request)
+	con, err := NewLDAPConnectAndBind(x.config)
+	defer con.Close()
+	if err != nil {
```

Comment:
I'm pretty sure you need to do the error checking before the `con.Close()`, because if the bind fails, then `con` will most likely be null, in which case you'll end up with a panic. There needs to be a unit test for this case.